pkg/scalers/splunk_o11y_scaler.go

```diff
@@ -0,0 +1,264 @@
+package scalers
+
+import (
+	"context"
+	"fmt"
+	"math"
+
+	//"os"
```

Comment:
Remove commented out imports, please :)

Reply:
Yes, I will do. Thank you for pointing this out to me.